commitizen/commands/changelog.py

```diff
@@ -55,8 +58,8 @@ def __init__(self, config: BaseConfig, args):
             or defaults.change_type_order
         )
         self.rev_range = args.get("rev_range")
-        self.tag_format = args.get("tag_format") or self.config.settings.get(
-            "tag_format"
+        self.tag_format: str = args.get("tag_format") or self.config.settings.get(
+            "tag_format", DEFAULT_SETTINGS["tag_format"]
```

Comment:
You don't need to inject default there, default are already loaded and overwritten by actual settings